src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php

```diff
@@ -53,9 +53,45 @@ class Connection
      */
     private $amqpQueue;
 
-    public function __construct(array $connectionCredentials, array $exchangeConfiguration, array $queueConfiguration, AmqpFactory $amqpFactory = null)
+    /**
+     * @var \AMQPExchange|null
+     */
+    private $amqpDelayExchange;
+
+    /**
+     * Available options:.
```

Comment:
extra dot at the end.

That's one case where having a generic `Constructor` as the first line is kind of required (if not, `Available options` is the method description).